apps/contacts/contacts.app.js

```diff
@@ -17,35 +17,49 @@ function callNumber (number) {
     categories:["android.intent.category.DEFAULT"],
     data: 'tel:' + number,
   }))
-
 }
 
 function mainMenu() {
   var menu = {
+    "": {
+      "title": "Contacts",
+    },
     "< Back" : Bangle.load
   };
-  if (!wp.length) {
+  if (!contacts.length) {
     menu['No Contacts'] = () => {};
-  } else {
-    for (const e of wp) {
-      const closureE = e;
-      menu[e.name] = () => showContact(closureE);
-    }
   }
-  menu["Add"] = addContact;
-  menu["Remove"] = removeContact;
+  contacts.map((e, idx) => {
```

Comment:
You can use `.forEach` if you don't use the result of `map`, so we avoid creating a new array :)

```suggestion
  contacts.forEach((e, idx) => {
```